<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtProxyClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="msgEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMSG">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="recvLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Received Data</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msgLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="ipEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;127.0.0.1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ip_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Desitination IP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="recvData">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearRECV">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>93</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;8088&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="port_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Desitination Port</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>93</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runTestSuite">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>93</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Test Suite</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="testPathEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectTestSuite">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>93</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Test Suite</string>
    </property>
   </widget>
   <widget class="QLabel" name="test_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Path</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="testProgress">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="testOut">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="test_Label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
